README.md

```diff
@@ -1,103 +1,125 @@
-## Web Monetization Extension
+<h1 align="center">Web Monetization Extension</h1>
 
 ![Github Actions CI](https://github.com/interledger/web-monetization-extension/actions/workflows/sanity.yml/badge.svg?branch=main)
 [![Prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://prettier.io/)
 
+</center>
 Web Monetization is a browser extension that detects Web Monetization on websites using a browser API that allows the
```

Comment:
Suggesting rewording: 
The Web Monetization browser extension is an open source implementation of the Web Monetization draft specification - learn more [here](https://webmonetization.org/specification/). The extension is built with React and TypeScript.

This Web Monetization browser extension: 
* Detects whether a user is focused on a website that is Web Monetization enabled. 
* Uses a browser API to create a payment stream from the authorized digital wallet or account of a user, to the website.